ArduPlane/Parameters.cpp

```diff
@@ -205,6 +205,24 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Advanced
     GSCALAR(takeoff_pitch_limit_reduction_sec, "TKOFF_PLIM_SEC",  2),
 
+    // @Param: TKOFF_RDDR_FF
+    // @DisplayName: Takeoff aileron to Rudder feedforward
+    // @Description: This parameter sets aileron to rudder ff below the TKOFF_RDDR_FF_H hieght, helpful to resolve tendency of plane to roll during takeoff.
+    // @Units: fraction
```

Comment:
This isn't the known units - it can be added, but I don't think fraction is a good, understandable unit. In any case, I think this and KFF_RUDDER_MIX should have the same unit (the latter is unit-less now).